core/src/main/java/io/substrait/relation/RelProtoConverter.java

```diff
@@ -75,6 +77,43 @@ private List<SortField> toProtoS(Collection<Expression.SortField> sorts) {
         .collect(java.util.stream.Collectors.toList());
   }
 
+  private List<ConsistentPartitionWindowRel.WindowRelFunction> toProtoWindowRelFunctions(
```

Comment:
minor: Can you move this to directly after the definition of `public Rel visit(ConsistentPartitionWindow` as it's the only caller of this.

Reply:
Ack